string img_path

---------------
bool success
string[] message
